src/models/.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'mongoose';
```

Comment:
I don't think type declarations are needed here. The @types/mongoose library is already installed so you just need to go to tsconfig and list it under compilerOptions/types

Reply:
OK, removing this file